dist/index.html

```diff
@@ -6,6 +6,5 @@
     <title>Wbpack Exercise</title>
 <script defer src="main.js"></script></head>
 <body>
-    <h1>Hello webpack!</h1>
```

Comment:
- [ ] Kindly update the readme.md following [this template](https://github.com/microverseinc/readme-template).